demosplan/DemosPlanCoreBundle/ResourceTypes/GisLayerCategoryResourceType.php

```diff
@@ -21,13 +21,15 @@
  * @template-extends DplanResourceType<GisLayerCategory>
  *
  * @property-read End $name
+ * @property-read End $isRootCategory
```

Comment:
@sierrazamudiodemos i think it has to be removed

Reply:
@sakutademos i removed it, you can pull